yolo_to_yolo/test_bbox_to_crop_transformer.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
```

Comment:
I was thinking to have the test script run on some of our actual data, but this is good enough. I did realize that you don't have access to our data yet.